engine/src/main/java/org/terasology/recording/RecordAndReplayStatus.java

```diff
@@ -22,13 +22,14 @@
     NOT_ACTIVATED, PREPARING_RECORD, RECORDING, PREPARING_REPLAY, REPLAYING, REPLAY_FINISHED;
 
     /** Status of the Record and Replay in the current game */
+    /*
```

Comment:
Might want to either get rid of commented out code entirely or leave a `TODO` describing what's pending to re-enable